src/types/index.d.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable camelcase */
+import type { GridLayerOptions, Coords, DoneCallback } from "leaflet";
+
+export type PixelValueToColorFn = (values: number[]) => string;
+
+export interface GeorasterLayerOptions extends GridLayerOptions {
+  georasters: Georaster[];
+  georaster: Georaster;
+  resolution?: number;
+  debugLevel?: 0 | 1;
```

Comment:
This should be `debugLevel?: 0 | 1 | 2`

Reply:
Sorry I haven't been consistent in the source code and sometimes do if (debugLevel > ...) and sometimes (debugLevel >= ... )

Reply:
Good to know. No worries.